chart/mqtt-connector/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenFaaS MQTT Connector chart
+
+## Installation
+
+```sh
+git clone https://github.com/openfaas-incubator/mqtt-connector
+cd mqtt-connector/chart
+
+# Template and apply:
+helm template -n openfaas --namespace openfaas mqtt-connector/ | kubectl apply -f -
+```
+
+You can watch the Connector logs to see it invoke your functions:
+
+```
+kubectl logs deployment.apps/openfaas-mqtt-connector -n openfaas -f
+```
+
+## Configuration
+
+Configure via `values.yaml`.
+
+| Parameter                | Description                                                               | Example                                     |
+| ------------------------ | ------------------------------------------------------------------------- | ------------------------------------------- |
```

Comment:
If these values have defaults, i think the readme should show those too, this can be very helpful for people trying to understand what to expect when deployed.

Reply:
@viveksyngh did you see this comment? 

If you're copying from my original MQTT chart (which I would expect you to do), then there is a README already with values you can copy and paste:

https://github.com/openfaas-incubator/mqtt-connector/tree/master/chart/mqtt-connector

Reply:
Makes sense. @viveksyngh can you cover in a follow-up PR and change from `Example` to `Default value`?

Reply:
I will do that.